js/script.js

```diff
@@ -37,16 +37,68 @@ const deleteTaskHandler = (event) => {
     taskList = taskList.filter((task) => task.id!==taskId);
 };
 
+const updateTaskEditHandler = (task, $taskDetails, $inputField, $editButton, $updateButton, $cancelButton) => {
+    task.title = $inputField.value;
+    
+    taskList.map((t) => {
```

Comment:
Why are we using `map` here? As its return a new array. We can use `find` instead.